stl/inc/xtree

```diff
@@ -1250,19 +1250,31 @@ public:
         return iterator(_Emplace_hint(_Where._Ptr, _STD move(_Val)), _Scary);
     }
 
+protected:
+    template <class _Iter, class _Sent>
+    void _Insert_range_unchecked(_Iter _First, const _Sent _Last) {
+        auto _Hint = _Get_scary()->_Myhead;
+        for (; _First != _Last; ++_First) {
+            _Hint = _Emplace_hint(_Hint, *_First);
+        }

```

Comment:
The new strategy seems problematic. Perhaps we should restore the old one.
```suggestion
        const auto _Myhead = _Get_scary()->_Myhead;
        for (; _First != _Last; ++_First) {
            _Emplace_hint(_Myhead, *_First);
        }
```

Reply:
Yeah, I reverted this. Apparently `_Emplace_hint` doesn't work the way I assumed.